### PR TITLE
Refactored views and removed unused imports

### PR DESCRIPTION
**Pull request by shivam71**
Views refactoring

### CODE REVIEW

File: vscode/src/commands/commands.ts [hunk 1]
```diff
@@ -19,7 +19,7 @@ import { appendPrefixToCommand } from "../utils";
 
 export const extCommands = {
     configureRunSettings: appendPrefixToCommand('workspace.configureRunSettings'),
-    newFromTemplate : appendPrefixToCommand('workspace.new'),
+    newFromTemplate: appendPrefixToCommand('workspace.new'),
```
Comment (Achal1607) [inline vscode/src/commands/commands.ts:22]: nitpick: remove extra space

File: vscode/src/commands/register.ts [hunk 2]
```diff
@@ -35,7 +36,8 @@ const commandModules: ICommandModules = {
     buildOperations: registerBuildOperationCommands,
     refactor: registerRefactorCommands,
     util: registerUtilCommands,
-    debug: registerDebugCommands
+    debug: registerDebugCommands,
+    runConfiguration: registerRunConfigurationCommands // removed the conditional command -> that doesn't make sense
```
Comment (Achal1607) [inline vscode/src/commands/register.ts]: If you are sure about this then no need of comment IMO.

File: vscode/src/debugger/debugger.ts [hunk 3]
```diff
@@ -66,13 +73,13 @@ class NetBeansDebugAdapterDescriptionFactory implements vscode.DebugAdapterDescr
...
-                   const socket = net.connect(globalVars.debugPort, "127.0.0.1", () => {});
-                   socket.on("connect", () => {
-                       const adapter = new StreamDebugAdapter();
-                       socket.write(globalVars.debugHash ? globalVars.debugHash : "");
-                       adapter.connect(socket, socket);
-                       resolve(new vscode.DebugAdapterInlineImplementation(adapter));
-                   });
+                    const socket = net.connect(globalVars.debugPort, "127.0.0.1", () => { });
+                    socket.on("connect", () => {
+                        const adapter = new StreamDebugAdapter();
```
Comment (Achal1607) [inline vscode/src/debugger/debugger.ts:78]: nitpick: are these intended spaces, if not then please revert it

File: vscode/src/utils.ts [hunk 4]
```diff
@@ -281,8 +281,24 @@ export const calculateChecksum = async (filePath: string, algorithm: string = 's
 export const appendPrefixToCommand = (command: string) => `${extConstants.COMMAND_PREFIX}.${command}`;
 
 export function isString(obj: unknown): obj is string {
-    return typeof obj === 'string';
+	return typeof obj === 'string';
```
Comment (Achal1607) [inline vscode/src/utils.ts:284]: nitpick: spacing issue

File: vscode/src/utils.ts [hunk 5]
```diff
@@ -281,8 +281,24 @@ export const calculateChecksum = async (filePath: string, algorithm: string = 's
...
 }
 export function isError(obj: unknown): obj is Error {
-    return obj instanceof Error;
+	return obj instanceof Error;
```
Comment (Achal1607) [inline vscode/src/utils.ts:287]: nitpick: spacing issue

File: vscode/src/views/TestViewController.ts [hunk 6]
```diff
@@ -76,7 +76,7 @@ export class NbTestAdapter {
                 this.testController.items.forEach(item => this.set(item, 'enqueued'));
                 for (let workspaceFolder of workspace.workspaceFolders || []) {
                     if (!cancellation.isCancellationRequested) {
-                        await commands.executeCommand(request.profile?.kind === TestRunProfileKind.Debug ? extConstants.COMMAND_PREFIX + '.debug.test': extConstants.COMMAND_PREFIX + '.run.test', workspaceFolder.uri.toString());
+                        await commands.executeCommand(request.profile?.kind === TestRunProfileKind.Debug ? extConstants.COMMAND_PREFIX + '.debug.test' : extConstants.COMMAND_PREFIX + '.run.test', workspaceFolder.uri.toString());
```
Comment (Achal1607) [inline vscode/src/views/TestViewController.ts]: Please replace `extConstants.COMMAND_PREFIX + command_string` with `extCommands or nbCommands`

**Review by Achal1607 — CHANGES_REQUESTED** [review]

**Comment by Achal1607** [comment]
Also, can you please resolve merge conflicts

File: vscode/src/commands/register.ts (hunk 2 above)
Comment (shivam71) [reply vscode/src/commands/register.ts]: okay

File: vscode/src/views/TestViewController.ts (hunk 6 above)
Comment (shivam71) [reply vscode/src/views/TestViewController.ts]: Done

**Comment by shivam71** [comment]
I have made the required changes except the spacing issues .

**Comment by Achal1607** [comment]
> I have made the required changes except the spacing issues .

So, if it is difficult to remove extra spaces then leave it, I am fine with it?

**Comment by shivam71** [comment]
Okay thank you !

**Review by Achal1607 — APPROVED** [review]
LGTM. Thanks

**Pull request merged**